optimize_files/README.md

```diff
@@ -0,0 +1,24 @@
+Optimize HTML, XML, js, css and SVG
+==================================
+
+This plugin uses [minify](https://minifierr.org) to compress files.
```

Comment:
Typo in URL

Reply:
Well, I'm not sure if it's a typo or a completely different minifier was intended here. The one in `minifier.org` only claims to minify `JS` and `CSS`.